src/main/java/create_task_use_case/CreateTaskController.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Create Task Controller Class.
+ * Author: Fardin Faruk
+ * Modified By: N/A
+ * Created: Nov 19, 2022
+ * Last Modified: N/A
+ */
+package create_task_use_case;
+
+public class CreateTaskController {
+
+    private final CreateTaskInputBoundary createTaskInput;
+
+    public CreateTaskController(CreateTaskInputBoundary createTaskInput) {
+        this.createTaskInput = createTaskInput;
```

Comment:
the "createTask" portion of createTaskInput is clear from its class. Also calling it an Input instead of InputBoundary could confuse this with some sort of input data.

Reply:
Sure, Done.